content/zh/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -765,14 +765,14 @@ to estimate or measure an optimal memory limit value for that container.
 ## {{% heading "whatsnext" %}}
 
 <!-- 
-* Learn about [API-initiated Eviction](/docs/concepts/scheduling-eviction/api-eviction/)
+* Learn about [API-initiated Eviction](/docs/reference/generated/kubernetes-api/v1.23/)
```

Comment:
Does this match the English original text? (It should be equivalent, but obviously in Chinese not English).